static/js/components/dashboard/CoursePrice.js

```diff
@@ -1,21 +1,69 @@
 // @flow
 import React from 'react';
 import _ from 'lodash';
+import ReactTooltip from 'react-tooltip';
+import IconButton from 'react-mdl/lib/IconButton';
+import moment from 'moment';
 
-import type { Course } from '../../flow/programTypes';
+import type { Course, CourseRun } from '../../flow/programTypes';
 import {
+  STATUS_NOT_OFFERED,
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
   STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_VERIFIED_NOT_COMPLETED,
   STATUS_OFFERED_NOT_ENROLLED,
+  DASHBOARD_FORMAT,
 } from '../../constants';
 import { formatPrice } from '../../util/util';
 
 export default class CoursePrice extends React.Component {
   props: {
     course: Course,
+    now: moment$Moment,
   };
 
+  help(course: Course, now: moment$Moment, firstRun: CourseRun): React$Element<*> {
```

Comment:
fwiw, in practice that tooltip is going to be used to explain personalized pricing -- for programs that offer personalized pricing. 

The original text about verified courses is going to be replaced.